api/v1/juliana.py

```diff
@@ -70,18 +72,23 @@ def juliana_rfid_get(request, event_id, rfid):
     return res
 
 
-@jsonrpc_method('juliana.order.save(Number,Number,Array) -> Nil', site=api_v1_site, authenticated=True)
+@jsonrpc_method('juliana.order.save(Number,Number,Array,Number) -> Nil', site=api_v1_site, authenticated=True)
 @transaction.atomic
-def juliana_order_save(request, event_id, user_id, purchases):
+def juliana_order_save(request, event_id, user_id, purchases, rfid_id):
```

Comment:
Please rename the last parameter to `rfid` because it is not the numeric id of the Rfid Django object but a JSON object/dictionary like in `rfid_to_identifier` and `juliana_rfid_get`.